package.json

```diff
@@ -35,16 +35,15 @@
   "homepage": "https://github.com/ipfs/js-datastore-fs#readme",
   "dependencies": {
     "async": "^2.6.1",
-    "datastore-core": "~0.6.0",
+    "datastore-core": "git://github.com:zcstarr/js-datastore-core.git",
     "fast-write-atomic": "~0.2.0",
     "glob": "^7.1.3",
     "graceful-fs": "^4.1.11",
-    "interface-datastore": "~0.6.0",
-    "mkdirp": "~0.5.1",
-    "pull-stream": "^3.6.9"
+    "interface-datastore": "git://github.com/ipfs/interface-datastore.git#refactor/async-iterators",
```

Comment:
As the `refactor/async-iterators` PR was merged and the branch deleted this is outdated and breaks things.